<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>791</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>681</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>450</width>
        <height>350</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>450</width>
        <height>350</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:5px solid #242424</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>450</y>
        <width>671</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="styleSheet">
          <string notr="true">border:1px solid #242424</string>
         </property>
         <property name="text">
          <string>图片1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">border:1px solid #242424</string>
         </property>
         <property name="text">
          <string>图片2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">border:1px solid #242424</string>
         </property>
         <property name="text">
          <string>图片3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">border:1px solid #242424</string>
         </property>
         <property name="text">
          <string>图片4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">border:1px solid #242424</string>
         </property>
         <property name="text">
          <string>图片5</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="save">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
      </property>
      <property name="text">
       <string>保存图片</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="videoWidget" native="true">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>370</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>亮度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>440</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>对比度：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>370</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>色相：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>440</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>饱和度：</string>
      </property>
     </widget>
     <widget class="QSlider" name="brightnessSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="hueSlider">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>400</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="contrastSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>470</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="saturationSlider">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>470</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>330</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:3px solid #242424</string>
      </property>
      <property name="text">
       <string>视频调节：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pause_btn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
      </property>
      <property name="text">
       <string>暂停</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play_btn">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
      </property>
      <property name="text">
       <string>播放</string>
      </property>
     </widget>
     <widget class="QSlider" name="VideohorizontalSlider">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>401</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="test">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>80</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="label_widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>501</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_VideoWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>0</y>
         <width>500</width>
         <height>400</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>400</height>
        </size>
       </property>
       <property name="text">
        <string>视频播放</string>
       </property>
      </widget>
      <widget class="QPushButton" name="twoValue">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>430</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>阈值调节</string>
       </property>
      </widget>
      <widget class="QPushButton" name="video_Avg">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>430</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>滤波调节</string>
       </property>
      </widget>
      <widget class="QPushButton" name="video_gamma">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>430</y>
         <width>91</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>伽马调节</string>
       </property>
      </widget>
      <widget class="QPushButton" name="video_RGB">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>430</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>RGB值调节</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件（&amp;F）</string>
    </property>
    <addaction name="action_O"/>
    <addaction name="action_V"/>
   </widget>
   <widget class="QMenu" name="menu_I">
    <property name="title">
     <string>图像处理（&amp;I）</string>
    </property>
    <addaction name="action_G"/>
    <addaction name="action_T"/>
    <addaction name="action_L"/>
    <addaction name="action_Y"/>
    <addaction name="action_B"/>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视频播放(&amp;V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_C">
    <property name="title">
     <string>帮助(&amp;C)</string>
    </property>
    <addaction name="action_E"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_I"/>
   <addaction name="menu_V"/>
   <addaction name="menu_C"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_O"/>
   <addaction name="action_V"/>
   <addaction name="action_G"/>
   <addaction name="action_T"/>
   <addaction name="action_L"/>
   <addaction name="action_B"/>
   <addaction name="action_Y"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>700</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #242424</string>
     </property>
     <property name="text">
      <string>阈值调节：</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="pageStep">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="minimum">
      <number>3</number>
     </property>
     <property name="maximum">
      <number>15</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #242424</string>
     </property>
     <property name="text">
      <string>滤波调节:</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #242424</string>
     </property>
     <property name="text">
      <string>伽马值调节：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>R值：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>G值：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>380</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>B值：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_R">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_G">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>350</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_B">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>390</y>
       <width>100</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>700</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QPushButton" name="toGray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>灰度化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toBlack">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>二值化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toAvg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>3X3滤波</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toGamma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>伽马变换</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Border">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>边缘检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="return_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>原画</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cut">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:2px groove gray;border-radius:10px;padding:2px 4px;</string>
     </property>
     <property name="text">
      <string>截图</string>
     </property>
    </widget>
    <widget class="QLabel" name="show_cut">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>300</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #242424</string>
     </property>
     <property name="text">
      <string>showLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>保存的截图：</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="action_G">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/heibai.png</normaloff>:/images/heibai.png</iconset>
   </property>
   <property name="text">
    <string>灰度化(&amp;G)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_T">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/erzhihua.png</normaloff>:/images/erzhihua.png</iconset>
   </property>
   <property name="text">
    <string>二值化(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_L">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/lvbo.png</normaloff>:/images/lvbo.png</iconset>
   </property>
   <property name="text">
    <string>滤波(&amp;L)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="action_Y">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/gamma.png</normaloff>:/images/gamma.png</iconset>
   </property>
   <property name="text">
    <string>伽马变换(&amp;Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_O">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/filenew.png</normaloff>:/images/filenew.png</iconset>
   </property>
   <property name="text">
    <string>打开文件(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_V">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/video.png</normaloff>:/images/video.png</iconset>
   </property>
   <property name="text">
    <string>
打开视频(&amp;V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_B">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/bianyuanjiance.png</normaloff>:/images/bianyuanjiance.png</iconset>
   </property>
   <property name="text">
    <string>边缘检测(&amp;B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_E">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
   </property>
   <property name="text">
    <string>语言转换(&amp;E)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
